packages/ra-core/src/core/CoreAdminUI.tsx

```diff
@@ -218,18 +260,29 @@ export interface CoreAdminUIProps {
 }
 
 export const CoreAdminUI = (props: CoreAdminUIProps) => {
+    const [errorInfo, setErrorInfo] = useState<ErrorInfo | undefined>(
+        undefined
+    );
     const {
+        authCallbackPage: LoginCallbackPage = false,
         catchAll = Noop,
         children,
         dashboard,
         disableTelemetry = false,
+        error = ({ error }) => (
+            <div>
+                <p>Error: {error?.message}</p>
+                <p>ErrorInfo: {JSON.stringify(errorInfo)}</p>
+                <p>ComponentStack: {errorInfo?.componentStack}</p>
+            </div>
+        ),
```

Comment:
Move it to a const in the enclosing scope

Reply:
done